### ISSUE TITLE
License for events from "Münsterland e.V." ?

### ISSUE THREAD

**Issue by ThorbenJensen** [codeformuenster/muenster-jetzt#221]
Currently using "NULL" license as a placeholder.

> I'm trying to find out what license they are using, but didn't get a response yet. I'll try calling them tomorrow. If I dont find out anything, I'll set a null value.

_Originally posted by @tomsrocket in https://github.com/codeformuenster/muenster-jetzt/pull/187#discussion_r490338449_

**Comment by tomsrocket** [comment]
I'm still on it! Only two persons can answer this. Person a) ist on vacation, Person b) is busy until wednesday. 🤷

**Comment by tomsrocket** [comment]
"Die Daten entsprechen nach Creative_Commens der Lizensierung CC-BY-SA, also mit Namensnennung."

**Comment by ubergesundheit** [comment]
@tomsrocket Any progress here?

**Comment by tomsrocket** [comment]
TLDR: It's not so easy. We can just use the data and refer to the source Münsterland e.V. somewhere. We should leave the license field empty for now.

Long Story: There is no standard license yet. They sent me a several page PDF file with "Nutzungsbedingungen". Reason is, this is a project that is still in the build-up phase and the license has to be aligned with many players, which is still in progress. Furthermore, right now it is planned to have license-fields within the data-feed e.g. so that every image could have a different license. That is also not finished yet.
